src/models/hit-types/hit.ts

```diff
@@ -8,16 +8,19 @@ import type { TextHit } from './text-hit';
 export type HitType = 'item' | 'text';
 
 /**
- * Applying this to the Result type forces Intellisense to present Result as
- * a type in its own right, and not as the underlying merge type it aliases.
- * Really just to keep things clean at the call site.
+ * Additional information provided by the PPS about hits, separately from
+ * their fields map.
  */
-interface PreserveAlias {} // eslint-disable-line @typescript-eslint/no-empty-interface
+interface HitInfo {
+  hit_type?: HitType;
+  index?: string;
+  service_backend?: string;
+}
```

Comment:
An imminent PPS upgrade (allowing multiple hit types in one response) will add these properties to hits. Bonus: we no longer need this little type-hack from before.